images/base/files/etc/containerd/cri-base.json

```diff
@@ -0,0 +1,237 @@
+{
+  "ociVersion": "1.0.2-dev",
```

Comment:
we're presumably going to need to generate this again when containerd updates?
we should perhaps script that somewhere?

Reply:
In that case maybe is better to go back to generating cri-base.json at the base image Dockefile so we have always the correct config.

Reply:
or do you mean to script it here?

https://github.com/kubernetes-sigs/kind/blob/04dff0f67d632e23a868cf86bda37bee87b2df6c/hack/make-rules/update/all.sh#L22-L22

we are controlling the containerd version in kind

https://github.com/kubernetes-sigs/kind/blob/e05318fbfa3aaac2773ce1f2b828afa135fc9fb2/images/base/Dockerfile#L31-L31

Reply:
@aojea I don't see the update task being called inside the base image container Dockerfile

Reply:
@aojea so the alternative is to parse the Dockerfile to extrant the CONATINERD_VERSION variable and then download the ctr tool to generate the cri-base.json and patch to add the hook, that's right ?

Reply:
puff, Ben is more familiar than me with this glue, if he is not able to respond during this week I'll take a look,

Reply:
@BenTheElder @aojea I think is better to go back to generate the cri-base.json at the Dockerfile, patch with jq and uninstall jq (It's no longer needed to get the bundle path), what do you think ?

Reply:
I think it's probably best to do that, it has no impact on the image size and it should be easier to keep the base up to date that way I think.

Reply:
Done